include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1620,6 +1619,81 @@ namespace internal
 } // namespace internal
 
 
+
+namespace MGTransferGlobalCoarseningTools
+{
+  template <int dim, int spacedim>
+  std::vector<std::shared_ptr<Triangulation<dim, spacedim>>>
+  create_geometric_coarsening_sequence(
+    const Triangulation<dim, spacedim> &fine_triangulation_in)
+  {
+    std::vector<std::shared_ptr<Triangulation<dim, spacedim>>>
+      coarse_grid_triangulations;
+
+#ifndef DEAL_II_WITH_P4EST
+    (void)fine_triangulation_in;
```

Comment:
You may want to throw an assertion here as well since this hasn't been implemented for any other Triangulation besides the `p::d` one.